src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/chat-message-editor.ts

```diff
@@ -109,6 +124,9 @@ export class ChatMessageEditor {
         }
         this.attachmentListObserver.disconnect();
         this.notifyPanelObserver.disconnect();
+        this.sideNavs.forEach(_ => {
```

Comment:
Why do you need forEach here? You can invoke `this.sideNavObserver.disconnect()` once.

Reply:
I'm stupid. I'll fix it.